brother_printer_fwupd/firmware_downloader.py

```diff
@@ -15,7 +16,8 @@
 
 
 def get_download_url(
-    printer_info: "SNMPPrinterInfo", firmid: str = "MAIN"
+    printer_info: "SNMPPrinterInfo", firmid: str = "MAIN",
+    reported_os: typing.Optional[str] = None
```

Comment:
Please run `black` over all files to format the code 😉

Reply:
Can do, but note that there are other pre-existing parts of the program which I _haven't_ changed, which are also not `black`-compliant. That's why I didn't run it already, because I wasn't sure if you want everything `black`ified.

Reply:
Oops. I'll add a `pre-commit` config after merging your PR 😉